internal/codespaces/api/api.go

```diff
@@ -240,6 +240,8 @@ const (
 	CodespaceStateAvailable = "Available"
 	// CodespaceStateShutdown is the state for a shutdown codespace environment.
 	CodespaceStateShutdown = "Shutdown"
+	// CodespaceStateShuttingDown is the state for a shutting down codespace environment.
```

Comment:
I'm kind of shocked this case wasn't already handled!

Reply:
Same!